### PR TITLE
Fix/add types and interfaces in components

### PR DESCRIPTION
**Pull request by jitprosen360**
Component files adding types and interfaces

### CODE REVIEW

**Review by andrew-bierman — CHANGES_REQUESTED** [review]
please rebase or copy changes to latest file structure

File: client/app.example.json
Comment (andrew-bierman) [inline client/app.example.json:1]: don't delete example file

Comment (jitprosen360) [reply client/app.example.json:1]: solved

**Comment by andrew-bierman** [comment]
these should be going into the /packages/app directory now, not /client directory. copy these changes to a clean branch off of andrew_testing please

**Pull request closed without merging**